zcash_client_backend/src/zip321.rs

```diff
@@ -152,6 +152,21 @@ impl TransactionRequest {
         &self.payments[..]
     }
 
+    /// Returns the total value of payments to be made.
+    ///
+    /// Returns `Err` in the case of overflow, if payment values are negative, or the value is
+    /// outside the valid range of Zcash values. .
```

Comment:
```suggestion
    /// outside the range `0..=MAX_MONEY` zatoshis.
```
"The valid range of Zcash values" is ambiguous.